### PR TITLE
fix: implement field for button

### PR DESCRIPTION
**Pull request by chungnguyenhoang**
## Check List
- [ ] Your code builds clean without any errors or warnings.
- [ ] Check coding convention.
- [ ] Updated Storybook components and stories to reflect any changes made to the UI.
- [ ] Test covers all edge cases and coverage is greater than or equal to 80%.

## Proof of Completeness
Screenshot changes introduced" width="250" />
OR
Video link

### CODE REVIEW

**Review by honghoangsts — APPROVED** [review]

File: src/components/Button/Button.tsx [hunk 1]
```diff
@@ -101,11 +108,17 @@ const ButtonWrapper = styled.TouchableOpacity(
...
+    leftIcon,
+    rightIcon,
   }: Omit<ButtonProps, 'text' | 'onPress'> & {theme?: ITheme}) => ({
     paddingVertical: theme?.spacing.small,
     paddingHorizontal: theme?.spacing.slim,
     borderRadius,
     backgroundColor: disabled ? theme?.colors.muted : backgroundColor || theme?.colors.green,
+    ...((leftIcon || rightIcon) && {
+      alignItems: 'center',
+      fexDirection: 'row',
```
Comment (honghoangsts) [inline src/components/Button/Button.tsx:120]: typo: `flexDirection`
should we make it row as default?

**Review by honghoangsts — CHANGES_REQUESTED** [review]

File: src/components/Button/Button.tsx [hunk 2]
```diff
@@ -101,11 +108,17 @@ const ButtonWrapper = styled.TouchableOpacity(
     outlineColor,
     borderRadius,
     disabled,
+    leftIcon,
+    rightIcon,
   }: Omit<ButtonProps, 'text' | 'onPress'> & {theme?: ITheme}) => ({
     paddingVertical: theme?.spacing.small,
+    fexDirection: 'row',
```
Comment (honghoangsts) [inline src/components/Button/Button.tsx:115]: `fexDirection`?

**Review by honghoangsts — CHANGES_REQUESTED** [review]

**Review by honghoangsts — APPROVED** [review]
